tests/integration/targets/version_update/tasks/main.yml

```diff
@@ -24,9 +24,25 @@
     - name: Get status of the latest update applied
       scale_computing.hypercore.version_update_status_info:
       register: update_status
+    # Fail early if update is in progress
+    # In some cases update fails and we get record.update_status == None
+    # Example from https://172.31.6.11/update/update_status.json
+    # {
+    #    "prepareStatus": "",
+    #    "updateStatus": {
+    #        "percent": "0",
+    #        "status": {
+    #            "usernotes": "Failed to open update tunnel"
+    #        }
+    #    }
+    #}
     - ansible.builtin.assert:
         that:
-          - update_status.record.update_status == "COMPLETE" or update_status.record.update_status == "TERMINATING" # Fail early if update in progress
+          - >-
+            update_status.record == None or
+            update_status.record.update_status == None or
+            update_status.record.update_status == "COMPLETED" or
```

Comment:
Here should be COMPLETE (but I agree, "completed" sounds better :))

Reply:
A, one more typo. Indeed, I see this `update_status.json` content:
```
{
    "prepareStatus": "",
    "updateStatus": {
        "masterStateID": "4",
        "masterState": "COMPLETE",
        "fromBuild": "208456",
        "toBuild": "211326",
        "toVersion": "9.1.24.211326",
        "currentComponent": "728",
        "totalComponents": "728",
        "percent": "100",
        "status": {
            "component": "7690\/9999",
            "node": "172.31.7.11",
            "statusdetails": "Update Complete. Press 'Reload' to reconnect",
            "usernotes": "Update Complete. Press 'Reload' to reconnect"
        }
    }
}
```

Testing this is just to hard...